src/pages/StaticPages/PrivacyNotice.tsx

```diff
@@ -0,0 +1,316 @@
+import React, { useEffect } from 'react';
```

Comment:
Why do we have two separate Privacy Notice .tsx files: Privacy notice and privacy notice page .tsx?

Reply:
Removed privacy notice page, was just used for testing im assuming